packages/experimental/Avatar/src/JSAvatar.helpers.ts

```diff
@@ -106,21 +106,22 @@ export function calculateEffectiveSizes(tokens: JSAvatarTokens): AvatarSizeConfi
   }
 }
 
-export function getRingThickness(thickness: RingThickness): number {
-  switch (thickness) {
-    case 'xSmall':
-      return 1;
-    case 'small':
-      return 1;
-    case 'medium':
-      return 2;
-    case 'large':
-      return 2;
-    case 'xlarge':
-      return 2;
-    case 'xxlarge':
-      return 4;
-    default:
-      return thickness < 0 ? 4 : thickness;
-  }
+export function getRingConfig(size: number): any {
+  if (size <= 48)
+    return {
+      size: size + 8,
+      stroke: globalTokens.stroke.width.thick,
+      innerStroke: globalTokens.stroke.width.thick,
+    };
+  if (size <= 71)
+    return {
+      size: size + 12,
+      stroke: globalTokens.stroke.width.thicker,
+      innerStroke: globalTokens.stroke.width.thicker,
+    };
+  return {
+    size: size + 16,
```

Comment:
Can we move out the magic numbers here to their own variables?

Reply:
done